rotkehlchen/tests/api/test_balances.py

```diff
@@ -911,3 +912,35 @@ def test_balances_behaviour_with_manual_current_prices(rotkehlchen_api_server, e
         rdn_result = result['assets']['eip155:1/erc20:0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6']
         assert rdn_result['amount'] == '5'
         assert rdn_result['usd_value'] == '150.0'
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [1])
+def test_blockchain_balances_refresh(
```

Comment:
this test is doing remote calls right? Wouldn't it make sense to vcr and use a fixed address?